client/packages/common/src/authentication/hooks/index.ts

```diff
@@ -1,2 +1 @@
 export * from './usePermissionCheck';
-export * from './useCheckPermissionWithError';
```

Comment:
this hook isn't required, I think - and could cause confusion with the existing hook